runtime/layouts/microbit_v2.ld

```diff
@@ -0,0 +1,11 @@
+/* Layout for the micro:bit v2 board, used by the examples in this repository. */
+
+MEMORY {
+  FLASH (rx) : ORIGIN = 0x00040000, LENGTH = 256K
+  RAM (rwx) : ORIGIN = 0x20004000, LENGTH = 112K
+}
+
+MPU_MIN_ALIGN = 8K;
```

Comment:
`MPU_MIN_ALIGN` is not necessary here (`libtock_layout.ld` does not use it).

Reply:
Fixed